src/index.html

```diff
@@ -1,57 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="stylesheet"
-      href="https://pro.fontawesome.com/releases/v5.15.1/css/all.css"
-    />
-    <link rel="stylesheet" href="style.css" type="text/css" />
-    <title>To-Do-List</title>
+    <meta charset="UTF-8">  
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link href="style.css" rel="stylesheet">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Comment:
- [  ]  Please I can see that the user can edit a task but the issue is when the user edits two tasks a weird behavior happen one of the edited tasks seems to override editing the other one please fix this issue
![image](https://user-images.githubusercontent.com/74744787/185767612-a4f165e4-7607-4b83-bfae-dc631c5c0991.png)